docs/troubleshooting.md

```diff
@@ -1,29 +1,25 @@
 
 # Troubleshooting
 
-## I cannot join a worker node
+## Worker nodes cannot be joined
 
-In [standalone](usage-overview.md#standalone) mode, check whether the join
+If you are using k0smotron in standalone mode, check whether the join
```

Comment:
We need to provide some step how to inspect the token.

1. Decode the token:
```
echo "<token>" | base64 -d | gunzip
```
2. Take note of the field `users.user.token`. I looks something like `gb823t.b8ftcytc4ktmvkjz`
3. Using the KUBECONFIG of the child cluster:
```
kubectl -n kube-system get secret bootstrap-token-gb823t --template='{{.data.expiration}}' | base64 -d
2024-03-14T11:08:13Z
```

Reply:
@jnummelin, for Step 2, what will the user do with the `users.user.token` that they have taken note of? For now -- or, at least, for me -- this step just kind of hangs there.

Reply:
They need to take the first part, before the `.`, and use it as part of the secret name. In the above example it's the `gb823t `. So that maps the token to a secret in which the value is the actual full token information.